services/QuillLMS/client/app/bundles/Teacher/components/blog_posts/header_section.jsx

```diff
@@ -1,17 +1,30 @@
 import React from 'react'
 
-import { TEACHER_CENTER_SLUG } from './blog_post_constants'
+import { TEACHER_CENTER_SLUG, USING_QUILL_FOR_READING_COMPREHENSION } from './blog_post_constants'
 
-export default (props) => (
-  <div className="header-section">
-    <div className="text">
-      <h1>{props.title}</h1>
-      <p>{props.subtitle}</p>
+export const HeaderSection = ({ title, subtitle, showCancelSearchButton, query }) => {
+
+  function renderTitle() {
+    if(title === USING_QUILL_FOR_READING_COMPREHENSION) {
+      const words = title.split(" ");
+      const quill = words[1].charAt(0).toUpperCase() + words[1].slice(1);
+      return words[0] + ' ' + quill + ' ' + words.slice(2).join(" ");
```

Comment:
What does this produce? Maybe it's easier to just return a string literal?

Reply:
totally forgot you could do that! great suggestion, thank you